packages/terra-form-input/src/terra-dev-site/test/form-input/InputField.test.jsx

```diff
@@ -30,7 +30,7 @@ class InputFieldExample extends React.Component {
 
   render() {
     return (
-      <div>
+      <div style={{ caretColor: 'transparent' }}>
```

Comment:
We should not add this to our test examples directly. These examples are still functional verification candidate on PR reviews.